install

```diff
@@ -94,8 +94,12 @@ end
 def sudo?
   return @have_sudo unless @have_sudo.nil?
 
-  Kernel.system "/usr/bin/sudo", "-v"
-  @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success?
+  begin
+    Kernel.system "/usr/bin/sudo", "-l", "mkdir"
+    @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success? || false
```

Comment:
```suggestion
    @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success?
```
`nil` for `false` is fine.

Reply:
```suggestion
    @have_sudo = $CHILD_STATUS.success?
```
I was wrong here. `$CHILD_STATUS` is not `nil` when `/usr/bin/sudo` does not exist.

Reply:
Maybe I also did a similar misunderstanding that I observed `nil` before adding `require "English" in the test script...
The last one is enough, I agree.

I put `|| false` because if `@have_sudo` is nil, `sudo?` cannot pass the first line always.

Anyways, I updated to just `$CHILD_STATUS.success?`.

Reply:
> Maybe I also did a similar misunderstanding that I observed nil before adding `require "English" in the test script...

I think I made the identical error.

> I put || false because if @have_sudo is nil, sudo? cannot pass the first line always.

Yes, good point. I missed that.